patches/obs-glcapture.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/src/obs-glcapture b/src/obs-glcapture
+index 2efcf51..aecf1ef 100755
+--- a/src/obs-glcapture
++++ b/src/obs-glcapture
+@@ -1,6 +1,9 @@
+ #!/bin/sh
+ 
+ OBS_GLCAPTURE_LIB="libobs_glcapture.so"
++LIBDIR_BASE="/usr/lib/extensions/vulkan/OBSVkCapture/lib"
++LIBDIR_32="${LIBDIR_BASE}/i386-linux-gnu/"
++LIBDIR_64="${LIBDIR_BASE}/x86_64-linux-gnu/"
```

Comment:
It could be a bit cleaner to have a single arch-independent path with `$LIB` in it, e.g.
```
LIBDIR="/usr/lib/extensions/vulkan/OBSVkCapture/\$LIB"
```
(note that the escaping - `\$`)

Although it feels a bit like a hack, it seems to work for e.g. MangoHud.

Reply:
Didn't know you could do that but it works, thanks